uber/templates/preregistration/preregbase.html

```diff
@@ -95,7 +101,9 @@
                 price: {{ c.BADGE_TYPE_PRICES[badge_type] }},
                 badge_type: '{{ badge_type }}',
                 onClick: function () {
-                    $('input[name="badge_type"]').val('{{ badge_type }}');
+                    if ($('input[name="badge_type"]').val() != '{{ c.STAFF_BADGE }}') {
```

Comment:
Actually, can you take this change out? It won't affect MAGFest because we don't use these these kinds of badge buttons, but these buttons are meant to change badge types -- even if someone is staff.